ansible/userdata/cloud-config-etcd

```diff
@@ -70,15 +70,13 @@ coreos:
         [Install]
         WantedBy=multi-user.target
         
-{{if .DisableContainerLinuxAutomaticUpdates}}
```

Comment:
Should we put these in a different commit, as we definitely need them, as opposed to we're not sure with the k8s version update.

Reply:
I included this also in the PR: https://github.com/Financial-Times/content-k8s-provisioner/pull/263